package.json

```diff
@@ -38,7 +38,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged",
-      "pre-push": "npm test && npm audit && npm run remark"
+      "pre-push": "npm test"
```

Comment:
why ?

Reply:
I have put back the initial version. In the past I had some trouble to push with npm audit  and run remark. I didn't want to push this modfication. Now It works fine.